data_and_models/pipelines/sentence_embedding/Dockerfile

```diff
@@ -15,7 +15,7 @@
 # You should have received a copy of the GNU Lesser General Public License
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-FROM continuumio/miniconda3:4.9.2
+FROM nvcr.io/nvidia/pytorch:21.06-py3
```

Comment:
What is the difference with `nvidia/cuda:10.2-runtime` used in `bbs-base`?

Reply:
The difference is mainly that it is a `torch` official docker image.

Reply:
I see. Thanks!